multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2034,9 +2147,11 @@ def save(self, path: str, standalone: Optional[bool] = True):
                 fp,
                 ensure_ascii=True,
             )
+
         # In case that users save to a path, which is not the original save_path.
         if os.path.abspath(path) != os.path.abspath(self._save_path):
             model_path = os.path.join(self._save_path, "model.ckpt")
+            print(model_path, path)
```

Comment:
Remove `print()` or use `logger.debug()`.

Reply:
@Raldir I think we can change it.